h/static/scripts/controllers.coffee

```diff
@@ -170,45 +198,43 @@ class ViewerController
     loaded = []
 
     loadAnnotations = ->
```

Comment:
Again, this seems like an odd place for this function (and the `$watchCollection` below). It seems to me that this should be part of the responsibilities of `AnnotationSync`.